articles/141_static_remapping.md

```diff
@@ -339,8 +338,8 @@ Within each category, the rules are applied in the order in which the user gave
 **Example of node/namespace remapping order:**
 
 - A node has name `talker`
-- A user gives the rules `talker:__ns:=/my_namespace` then `talker:__node:=foo`
-- The final namespace is the default `/` because the node name remap is applied before the namespace remap
+- A user gives the rules `talker:__ns:=/my_namespace` then `talker:__node:=foo`, or vice versa
+- The final fully qualified name is `/my_namespace/foo` because the namespace and node name remap are applied without affecting each other
```

Comment:
>  the namespace and node name remap are applied without affecting each other

I would change this wording to: `the namespace and node name remap rules match against the orignal node name before remappings are applied.

Reply:
Thank you. I'll update it later.